release-plz.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
+dependencies_update = true                   # Whether to run `cargo update` in the release PR
+pr_name = "release: zksync-foundry {{version}}" # template for the PR name
+pr_labels = ["release", "automated"]         # Set PR labels
+changelog_update = false                     # Disable individual changelog updates for all packages.
+git_tag_enable = false                       # Disable individual tags for all packages.
+git_release_enable = false                   # Disable individual releases for all packages.
+semver_check = true                          # Enable API breaking changes checks with cargo-semver-checks.
```

Comment:
How is the semver check working?

Reply:
It's calling this tool: https://github.com/obi1kenobi/cargo-semver-checks

It checks for changes in your library that might inadvertently break semantic versioning rules. For example, removing a public function or changing its signature in a way that could break users relying on it.

The tool analyzes two versions of a Rust library by comparing the public API surface (functions, structs, traits, enums, etc.) of the two versions to detect breaking changes.

Reply:
We can disable it if we don't need it. CC: @dutterbutter

Reply:
That is sensible. I am fine with `0.2.0-zksync.1.0` or `<upstream>-zksync.<major>.<minor>`. I think dropping the `alpha` would be nice. 

Any concerns @antonbaliasnikov @nbaztec?

Reply:
Does it mean we pin the upstream version and update it manually in the config?

It would look like that then:
```
...
git_tag_name = "0.2.0-zksync.{{version}}" # Tag name
```

Where the `{{version}}` will be changed on our side accordingly to the conventional commits in the repository.

Reply:
@antonbaliasnikov 

> Where the {{version}} will be changed on our side accordingly to the conventional commits in the repository.

Yep. Thanks!